homeassistant/helpers/config_entry_flow.py

```diff
@@ -81,12 +81,6 @@ async def async_step_discovery(
 
         return await self.async_step_confirm()
 
-    async_step_zeroconf = async_step_discovery
```

Comment:
These need to be here because `async_step_discovery` logic needs to be assigned to all these discovery steps. It's not assigning the same thing.

Reply:
@balloob I have marked this as draft for now.
It was linked to PR 59503 - maybe you can take a look at this comment?
https://github.com/home-assistant/core/pull/59503#discussion_r748298035